skll/config.py

```diff
@@ -33,6 +35,19 @@
                                             'none'])
 
 
+def _transform_learner_name(learner_name):
```

Comment:
As I mention in a separate comment, I liked the idea of simply using `globals()` to do this and it would have made things a lot simpler and cleaner. However, it seems to me that `globals()` does not contain any entries that correspond to learner names. I can get access to learner names via `dir(skll.learner)`, so there is a way that I can confirm that a learner name is simply defined in `skll.learner`. However, to get the corresponding class, there's no obvious way (to me, at least) to do that without something like `eval('skll.learner.{}'.format(learner_name))`. While this probably isn't that dangerous (despite the use of `eval`) since it would be *after* confirming that the `learner_name` part is at least defined in `skll.learner`, I don't think it looks good or is any better than the way I did it by creating a new `_LEARNER_NAMES_TO_CLASSES` dict, etc.

Please correct me if I'm wrong.